1/src/components/ListItem.vue

```diff
@@ -41,24 +41,28 @@
         </span>
       </div>
       <div v-if="checkLoggedInUser">
-        <a v-if="this.todo.status" ><img title="Done" class="icon" src="../assets/done.png" /></a>
-        <a v-else @click="setTodoDone"><img title="Done_Pending" class="icon" src="../assets/done_pending.png" /></a>
+        <a v-if="this.todo.status">
+          <img title="Done" class="icon" src="../assets/done.png" />
+        </a>
+        <a v-else @click="setTodoDone">
+          <img title="Done_Pending" class="icon" src="../assets/done_pending.png" />
+        </a>
       </div>
```

Comment:
There is only indentation changes here, right? For your code-reviewer it's sometimes hard to see a difference in the implementation.